spec/factories/tickets.rb

```diff
@@ -7,4 +7,12 @@
     place { 5 }
     price { 10.0 }
   end
+
+  factory :ticket_sold, class: Ticket do
+    association :event, factory: :event
```

Comment:
Do tego można fajnie wykorzystać "traits" (https://devhints.io/factory_bot#other-features)

wtedy zamiast
`FactoryBot.create :sold_ticket`
byłoby
`FactoryBot.create :ticket, :sold`

traits to takie modyfikatory do factories, można ich dodawać wiele do jednego obiektu